package.json

```diff
@@ -15,7 +15,7 @@
     "Lu, Jianghao <jianghlu@microsoft.com>",
     "Krishnan, Balaji <Balaji.Krishnan@microsoft.com>"
   ],
-  "version": "2.3.1-preview",
+  "version": "2.3.2-preview",
```

Comment:
We control shipping the rollup version. We don't plan to ship the roll up version anytime soon. Please revert the change in this package.json file.

Reply:
Will do -- previously was told to bump this for Log Analytics so I did the same here. I'll revert.